### ISSUE TITLE
Shell module is not accepting " or ' in the command. 

### ISSUE THREAD

**Issue by baskaran-md** [apigee/henchman#123]
```
  - name: "Get Version of {{ item }}"
    shell: cmd="my-service {{ item }} version | cut -d ' ' -f2"
    debug: true
    with_items:
    - "cassandra"
    - "zookeeper"
```

**ERROR**

```
SHELL [ cmd => my-service cassandra version | cut -d   -f2 ]
failure => exec'ed command

Output
-----
stderr: cut: the delimiter must be a single character
Try 'cut --help' for more information.
echo: write error: Broken pipe

stdout:
```

**Comment by jlin21** [comment]
@sudharsh can you get this one?

**Comment by sudharsh** [comment]
This one is fixed with 3940eef44dafa8de7ea09d51ccaedc09cf3e7d78
